review_practice/api-server/config/database.js

```diff
@@ -0,0 +1,38 @@
+// import dotenv from 'dotenv';
+const dotenv = require('dotenv');
+dotenv.config();
+
+const {
+    DB_DEV_USER,
+    DB_DEV_PASSWORD,
+    DB_DEV_DATABASE,
+    DB_DEV_HOST,
+    DB_PRODUCTION_USER,
+    DB_PRODUCTION_PASSWORD,
+    DB_PRODUCTION_DATABASE,
+    DB_PRODUCTION_HOST,
+} = process.env;
+
+module.exports = {
+    development: {
+        username: DB_DEV_USER,
+        password: DB_DEV_PASSWORD,
+        database: DB_DEV_DATABASE,
+        host: DB_DEV_HOST,
+        dialect: 'mysql',
+    },
+    test: {
+        username: DB_DEV_USER,
+        password: DB_DEV_PASSWORD,
+        database: DB_DEV_DATABASE,
+        host: DB_DEV_HOST,
+        dialect: 'mysql',
+    },
```

Comment:
```suggestion

const DEV_ENV =  {
    username: DB_DEV_USER,
    password: DB_DEV_PASSWORD,
    database: DB_DEV_DATABASE,
    host: DB_DEV_HOST,
    dialect: 'mysql',
};

module.exports = {
    development: { ...DEV_ENV },
    test: { ...DEV_ENV },
```
이렇게 사용하면 확장하고 유지하기가 편리할 것 같아요!